intelmq/tests/bots/collectors/fireeye/test_collector.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import pathlib
+import secrets
+
+import requests_mock
+
+import intelmq.lib.test as test
+from intelmq.bots.collectors.fireeye.collector_fireeye import FireeyeCollectorBot
+
+RANDSTR = secrets.token_urlsafe(50)
+ASSET_PATH_FIRST = pathlib.Path(__file__).parent / '../../../assets/fireeyeFristRequest.json'
+ASSET_PATH_SECOND = pathlib.Path(__file__).parent / '../../../assets/fireeyeSecondRequest.xml'
+PARAMETERS = {'dns_name': 'myfireeye.local', 'http_username': RANDSTR, 'http_password': RANDSTR, 'logging_level': 'DEBUG', 'request_duration': '24_hours', 'name': 'FireeyeCollector'}
+
+
+def prepare_mocker(mocker):
+    mocker.post('https://myfireeye.local/wsapis/v2.0.0/auth/login', headers={'X-FeApi-Token': '1234567890'})
+    mocker.get('https://myfireeye.local/wsapis/v2.0.0/alerts?duration=24_hours', text=ASSET_PATH_FIRST.read_text())
+    mocker.get('https://myfireeye.local/wsapis/v2.0.0/openioc?alert_uuid=1591de22-4926-4124-b3ed-ffff96766295', text=ASSET_PATH_SECOND.read_text())
+
+
+@requests_mock.Mocker()
```

Comment:
Please add a `test.skip_exotic()` above this line. The same applies to the parser test. Then, the tests should also be working if the optional library xmltodict is not installed.